misc/module_config.json

```diff
@@ -1,10 +1,12 @@
 {
   "modules": {
     "pass": {
-      "enabled": true
+      "enabled": true, 
+      "timeout": 60
```

Comment:
What does this (time out on the "pass" module) do?  Pass should pretty much complete instantly?